internal/driver/constants.go

```diff
@@ -16,6 +16,8 @@ const (
 	AutoStreaming                   = "AutoStreaming"
 	InputIndex                      = "InputIndex"
 	UrlRawQuery                     = "urlRawQuery"
+	EnableRtspServer                = "EnableRtspServer"
+	RtspServerCmd                   = "./rtsp-simple-server"
```

Comment:
This should be made configurable. The executable path can vary depending on the deployment.